MarvellOS.py

```diff
@@ -126,26 +128,26 @@ def confirm(event=None):
     shbtn=Button(welcome_screen,text="Show",bg="light blue",fg="black",font=("Agency",8),command=show)
     shbtn.place(x=265,y=237)
 
-    confirmbtn=Button(welcome_screen,text="Confirm",bg="cyan",fg="black",font=("Arial black",9),command=confirm, width=20,height=2)                              #confirm button
-    confirmbtn.place(x=71,y=263)
+    confirmbtn=Button(welcome_screen,text="Confirm",bg="cyan",fg="black",font=("Arial black",10),command=confirm, width=14, height=2)                              #confirm button
+    confirmbtn.place(x=100,y=263)
```

Comment:
These are not the original values. Please look at the ones currently in the code.